lib/apkstats/command/apk_analyzer.rb

```diff
@@ -59,6 +73,15 @@ def self.to_feature(str)
       ::Apkstats::Entity::Feature.new(name, not_required: kind == "not-required", implied_reason: kind == "implied:" && tail)
     end
 
+    def self.parse_reference_to_map(command_output)
+      reference_map = {}
+      command_output.split(/\r?\n/).map do |s|
```

Comment:
This can be simplified like below.

```
reference_map = command_output.split(/\r?\n/).each_with_object({}) do |s, acc|
  dex_file, method_count = s...
  acc[dex_file] = method_count
end
```

Reply:
I don't know each_with_object method! Great.